### ISSUE TITLE
Issues with Data 

### ISSUE THREAD

**Issue by mlepori1** [deepskies/noise2self#2]
<img width="947" alt="Screen Shot 2019-07-08 at 11 58 50 PM" src="https://user-images.githubusercontent.com/25048682/61264280-95dd2b00-a751-11e9-91a8-12d30d6ea8e5.png">

Stumbled upon this while training the network.

**Comment by batson** [comment]
Check the range of values in each image when you see something weird like
this. I would modify plot_tensors to have a flag allowing you to clamp
images to zero below and also to plot all four with the same max intensity.
Grey in one image may be black in another because matplotlib by default
scales each image so min is black and max is white.

**Comment by bnord** [comment]
We may also want to be flagging these for removal before doing anything with them.
@mlepori1 are these nan's in the middle of the image?

**Comment by kadrlica** [comment]
@mlepori1 Are these being caused by the edge images that I documented in my note on slack? Are you applying the `flag` selection I described there? I'm guessing the gray region is `np.nan` values outside the edge of the exposure (you can see some real data on the left side).

**Comment by aruba19th** [comment]
<img width="341" alt="Screen Shot 2019-07-18 at 5 23 45 PM" src="https://user-images.githubusercontent.com/44248073/61497076-ffdd1680-a983-11e9-8fc1-94487087c631.png">
<img width="365" alt="Screen Shot 2019-07-18 at 5 23 27 PM" src="https://user-images.githubusercontent.com/44248073/61497086-04a1ca80-a984-11e9-9a55-f8d413a926a9.png">
I think now it's quite clear that the abnormally high pixel values in that thin line are causing such visualization in plots. I tried to exclude by examining the max pixel value. But there are certainly some normal-looking images that have extremely high maximum pixel value. I wonder if we should exclude these ones as well. Next, I will try to see their connection with flagging and try some new methods.
<img width="331" alt="Screen Shot 2019-07-18 at 5 24 34 PM" src="https://user-images.githubusercontent.com/44248073/61497500-39fae800-a985-11e9-926f-aa8f319fa9e3.png">

**Comment by kadrlica** [comment]
I don't think you can exclude images based on the max pixel values. There are several astronomical/detector artifacts that will create small numbers of very bright pixels. Throwing out an entire image due to small artifacts is unacceptable for the science. We may need to also send you masks...

**Comment by aruba19th** [comment]
Well, the original plan was to pick out the suspicious ones with max value and plot them. Then exclude them manually. But since there are so many of them, I am indeed thinking of other approaches.
